challengeutils/__main__.py

```diff
@@ -383,6 +384,22 @@ def command_add_mod_flair(syn, args):
     )
 
 
+def run_cheat_detection(syn, args):
+    """
+    Scan an evaluation queue for possible collaborators who are trying to exceed the submission limit.
+
+    >>> challengeutils cheat-detection <evaluation_id>
+    """
+
+    if type(args.submission_status) == str:
```

Comment:
It's more preferable in Python to do duck typing rather than actually get the type.  So a better alternative would be:
```suggestion
    if isinstance(args.submission_status, str):
```